cms/templates/bulkupdate.html

```diff
@@ -0,0 +1,101 @@
+<%inherit file="base.html" />
+<%!
+  from django.utils.translation import ugettext as _
+  from openedx.core.djangolib.js_utils import js_escaped_string
+%>
+<%block name="title">Utilities - Bulk Update</%block>
+<%block name="bodyclass">is-signedin course view-bulkupdate</%block>
+
+<%namespace name='static' file='static_content.html'/>
+
+
+<%block name="content">
+  <div class="wrapper-mast wrapper">
+    <header class="mast has-actions has-subtitle">
+      <h1 class="page-header">
+        <small class="subtitle">${_("Tools / Course Utilities")}</small>
+        <span class="sr">&gt; </span>${_("Bulk Update")}
+      </h1>
+    </header>
+  </div>
+
+  <div class="wrapper-content wrapper">
+    <section class="content">
+      <article class="content-primary" role="main">
+        <section class="course-utility" id="bulk-update">
+          <form id="bulk-update-settings-input">
+            <header class="section-header">Problem Settings</header>
+              <p>
+                <strong>Warning: </strong>
```

Comment:
Probably too late now, but for followup, the strings in this template should be i18n'ed.